server/container_execsync.go

```diff
@@ -16,13 +15,13 @@ func (s *Server) ExecSync(ctx context.Context, req *pb.ExecSyncRequest) (*pb.Exe
 		return nil, status.Errorf(codes.NotFound, "could not find container %q: %v", req.ContainerId, err)
 	}
 
-	if !c.IsAlive() {
-		return nil, fmt.Errorf("container is not created or running")
+	if err := c.IsAlive(); err != nil {
+		return nil, errors.Wrapf(err, "container is not created or running")
```

Comment:
Let's return the same `codes.NotFound` as the first check above.

Reply:
fixed!